dev-util/pycharm-community/pycharm-community-2024.3.5.ebuild

```diff
@@ -128,6 +136,27 @@ src_prepare() {
 		-e "\$a#-----------------------------------------------------------------------" \
 		-e "\$aide.no.platform.update=Gentoo" bin/idea.properties
 
+	# excepting files that should be kept for remote plugins
+	local skip_nonx86_files=(
+		"./plugins/platform-ijent-impl/ijent-aarch64-unknown-linux-musl-release"
```

Comment:
This should switch on amd64/arm64 and skip amd64+x86 versions on if installing on arm64 instead. Same for other versions and -professional

Reply:
> This should switch on amd64/arm64 and skip amd64+x86 versions if installing on arm64 instead. Same for other versions and -professional

I will take a look for supporting arm as well.

I have also seen your Clion PR and will analyse it when i have the time. Virtually it looks good, but I'll come back with recommendations for change.

Reply:
@WhatAmISupposedToPutHere based on your changes from CLion, I believe I will be supporting arm for pycharms as well, once CLion 2025.1-r1 will be merged.

@ceamac please do not merge these yet, I will support arm for these versions, and we can test thereafter.

Reply:
Mark it as draft until it's ready